akka-http/src/main/java/akka/http/javadsl/coding/Coder.java

```diff
@@ -25,9 +29,40 @@
         this.underlying = underlying;
     }
 
+    public HttpResponse encodeMessage(HttpResponse message) {
```

Comment:
This will not work, because `underlying.encodeMessage` needs a type that extends both `HttpMessage` and `MessageTranformations`. So what I could do is something like: `(HttpResponse) underlying.encodeMessage((SHttpResponse) response)`
Where `SHttpResponse` is the Scala one.